ProcessCIW/Models/CIW.cs

```diff
@@ -109,7 +110,11 @@ public string PersonalEmailAddress
 
         public string PositionJobTitle { get; set; }
         public string PriorInvestigation { get; set; }
-        public string ApproximiateInvestigationDate { get; set; }
+        public string ApproximiateInvestigationDate
```

Comment:
Please add comment that this is the prior investigation date.

Reply:
Comment added to identify Column name in database that corresponds with ApproximiateInvestigationDate.